### PR TITLE
benchmarking_deploy_coco_dataset: make the client-cert optional and download from upstream

### PR DESCRIPTION
**Pull request by kpouget**
Downloading from the private mirror is for nighly/repetitive testing,
for a one-shot download, no need to rely on our private mirror + its client cert.

Tested on my local cluster.

### CODE REVIEW

**Comment by kpouget** [comment]
@ArangoGutierrez can we merge this one?

**Comment by kpouget** [comment]
:facepalm:

/test test-commit

**Comment by kpouget** [comment]
no need for GPU Operator testing
/override ci/prow/gpu-operator-e2e

**Comment by kpouget** [comment]
mirror cluster isn't responding -,-
```
+ curl --cert /etc/mirror-secret/mirror.pem --silent --fail --show-error --retry 999999 --retry-max-time 1200 --continue-at - https://mirror-dataset.apps.ci-mirror.psap.aws.rhperfscale.org/coco/test2017.zip
+ tee test2017.zip
+ md5sum --check stdin.md5sum
curl: (35) OpenSSL SSL_connect: SSL_ERROR_SYSCALL in connection to mirror-dataset.apps.ci-mirror.psap.aws.rhperfscale.org:443 
-: FAILED
```
```
$ oc whoami
Unable to connect to the server: dial tcp 3.121.153.194:6443: i/o timeout
```

```
$ curl http://mirror-dataset.apps.ci-mirror.psap.aws.rhperfscale.org/coco/test2017.zip
curl: (52) Empty reply from server
```
retrying without using the mirror
/test test-commit

**Comment by kpouget** [comment]
/hold cancel

@ArangoGutierrez, can be merged this time :)

File: toolbox/benchmarking.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ def download_coco_dataset(node_hostname, namespace="default", pvc_name=None, mir
             namespace: Name of the namespace in which the resources will be created.
             pvc_name: Name of the PVC that will be create to store the dataset files.
             mirror_base_url: Optional base URL where to fetch the dataset
-            client_cert: Optional tath to the client cert to use for accessing the base URL.
+            client_cert: Optional path to the client cert to use for accessing the base URL.
```
Comment (ArangoGutierrez) [inline toolbox/benchmarking.py:19]: This could have been it's own commit

**Review by ArangoGutierrez — APPROVED** [review]
/lgtm

File: toolbox/benchmarking.py (hunk 1 above)
Comment (kpouget) [reply toolbox/benchmarking.py:19]: indeed, but it's the entrypoint of the role, so same topic 🙃

**Comment by kpouget** [comment]
> /lgtm

thanks @kpouget 

no need to test the GPU Operator, `test-commit` is enough
 /override ci/prow/gpu-operator-e2e

**Comment by kpouget** [comment]
bot is in vacation?

/joke

**Pull request merged**